pyiron_atomistics/sphinx/output_parser.py

```diff
@@ -0,0 +1,207 @@
+import numpy as np
+import re
+import scipy.constants
+from pyiron_base import state
```

Comment:
Can the `logger` be an input of the `SphinxLogParser` class, so that the `output_parser` module no longer depends on `pyiron_base` or `pyiron_atomistics`?

Reply:
That's an excellent point. Since it's not particularly important I can also think of using `warnings`. What do you prefer?

Reply:
I would prefer `warnings` that seems to be simpler from my perspective.